src/main/java/blusunrize/immersiveengineering/api/tool/ConveyorHandler.java

```diff
@@ -67,6 +72,36 @@
 	public static final Map<ResourceLocation, Block> conveyorBlocks = MetalDevices.CONVEYORS;
 	public static final ResourceLocation textureConveyorColour = new ResourceLocation("immersiveengineering:block/conveyor/colour");
 
+	// Should work for multiple dimensions since the calls aren't "interleaved" for multiple dimensions
+	private static final IntSet entitiesHandledInCurrentTickClient = new IntOpenHashSet();
+	private static final MutableLong currentTickClient = new MutableLong();
+	private static final IntSet entitiesHandledInCurrentTickServer = new IntOpenHashSet();
+	private static final MutableLong currentTickServer = new MutableLong();
+
+	// Returns true iff the entity has not been handled before in this tick
```

Comment:
Your if has a spare f =P

Reply:
No, it doesn't:
 > [if and only if (shortened as iff)](https://en.wikipedia.org/wiki/If_and_only_if)